CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+We are glad that you are interested in contributing to the Plasma Engine! We are happy to accept contributions, including:
+- bug reports
+- bug fixes
+- feature requests
+- new features
+- art assets
+- and much more!
+
+Please take a moment to read these guidelines to ensure your contributions are accepted.
+
+## Project Goals
+The goal of the H-uru Plasma project is to improve on the open source release of the CyanWorlds.com Engine used in Myst Online: Uru Live. In the interest of doing so, we have identified these project goals:
+- improving cross-platform and compiler compatibility
+- removing dependencies on nonfree libraries
+- fixing technical debt by utilizing modern language features in both C++ and Python
+- improving accessibility for fan Age creation and testing
+- improving testing coverage by adding unit tests and continuous integration
+- improving the player experience
+- avoiding nontrivial breaks in compatibility with the official Myst Online: Uru Live (again) game run by Cyan Worlds, Inc.
```

Comment:
Due to the unusual name, formatting such as "Myst Online: Uru Live (again)" or **Myst Online: Uru Live (again)** would make this easier to read.